docker/nginx/nginx.conf

```diff
@@ -36,11 +36,16 @@ http {
     include       /etc/nginx/mime.types;
     default_type  application/octet-stream;
 
+    map "$request:$status:$body_bytes_sent" $loggable {
```

Comment:
Cool can we do this in rero-ils?

Reply:
Its a long time waiting in https://github.com/rero/rero-ils/pull/677